projects/plugins/jetpack/extensions/blocks/recurring-payments/index.js

```diff
@@ -1,54 +1,41 @@
 /**
  * External dependencies
```

Comment:
This should be WordPress dependencies since you've deleted all the external deps.